packages/settings-library/src/settings_library/redis.py

```diff
@@ -15,6 +15,7 @@ class RedisDatabase(int, Enum):
     SCHEDULED_MAINTENANCE = 3
     USER_NOTIFICATIONS = 4
     ANNOUNCEMENTS = 5
+    DISTRIBUTED_IDENTIFIERS = 6
```

Comment:
For this case, I'm pretty confident the usage is correct. Since the Redis `KEYS` command is used to scan the entire keys-space, it is better to have these keys separate in a different DB.